src/util.h

```diff
@@ -387,6 +387,16 @@ constexpr size_t strsize(const T (&)[N]) {
 template <typename T, size_t kStackStorageSize = 1024>
 class MaybeStackBuffer {
  public:
+  // Disallow move constructor
+  MaybeStackBuffer(MaybeStackBuffer&&) = delete;
+  // Disallow copy constructor
+  MaybeStackBuffer(const MaybeStackBuffer&) = delete;
+  // Disallow move assignment operator
+  MaybeStackBuffer& operator=(MaybeStackBuffer&& other) = delete;
+  // Disallow copy assignment operator
+  MaybeStackBuffer&
+    operator=(MaybeStackBuffer& other) = delete;  // NOLINT(runtime/references)
+
```

Comment:
For brevity, deleting copy constructor and copy-assignment operator is sufficient.

- The [implicit move constructor](https://en.cppreference.com/w/cpp/language/move_constructor) is not defined if there is a user defined copy constructor,
- The [implicit move assignment operator](https://en.cppreference.com/w/cpp/language/move_assignment) is not defined if there is a user defined copy constructor.

```suggestion
  // Disallow copy constructor
  MaybeStackBuffer(const MaybeStackBuffer&) = delete;
  // Disallow copy assignment operator
  MaybeStackBuffer&
    operator=(const MaybeStackBuffer& other) = delete;

```